libcxx/include/__chrono/tzdb.h

```diff
@@ -31,6 +37,9 @@ namespace chrono {
 
 struct _LIBCPP_AVAILABILITY_TZDB tzdb {
   string version;
+  vector<pair<string, vector<__tz::__rule>>> __rules;
```

Comment:
What we really want here is `std::flat_map<string, vector<__tz::__rule>>`. I suggest we use `std::map` for now and switch it to `flat_map` once it is implemented. We could aim to have both in LLVM 19.

Reply:
Since we move the code to the dylib we can do this whenever we want. Since the code expects a vector and the move to the dylib loses test coverage I've only added a TODO and did not use `std::map` as a temporary measure.